### PR TITLE
enhance/chart-display

### PR DESCRIPTION
**Pull request by DmitriVanGuard**
### Summary
- Removed interval selector;
- Restrict all metrics (except price, volume, marketcap) to go lower than `1h` interval;
- "One day metrics" now have a day width bars;
- More metrics are moved to the `getMetric` api;

### Screenshots
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,7 +77,7 @@
     "react-transition-group": "^2.7.0",
     "react-virtualized": "^9.18.5",
     "react-virtualized-select": "^3.1.3",
-    "recharts": "^1.0.0-beta.10",
+    "recharts": "1.7",
```
Comment (GarageInc) [inline package.json]: Why 1.7? Current version is 1.8.5
https://github.com/recharts/recharts/blob/master/package.json

Comment (DmitriVanGuard) [reply package.json]: logarithmic scale is broken for bars

File: src/ducks/SANCharts/utils.js [hunk 2]
```diff
@@ -120,14 +118,51 @@ export const getMetricYAxisId = ({ yAxisId, key, dataKey = key }) => {
   return yAxisId || `axis-${dataKey}`
 }
 
+const alignDayMetrics = (chartRef, bars, dayMetrics, margin) => {
```
[the remark this answers is not in the document]
Comment (DmitriVanGuard) [reply src/ducks/SANCharts/utils.js]: I refactored this function a little bit.
Btw, do we really need to write a test for a few property accesses, object mutation and a ternary operator?

[the remark this answers is not in the document]
Comment (Partysun) [reply src/ducks/SANCharts/utils.js]: Any function can be described like this. But this func has 4 vars as input, counter vars inside and for in for, map construction. It is 100% enough function to make a mistake.

**Pull request merged**